apollo-compiler/src/main/kotlin/com/apollostack/compiler/FieldTypeSpecBuilder.kt

```diff
@@ -36,11 +36,19 @@ class FieldTypeSpecBuilder {
   /** Returns a list of types referenced by the inner fields in the provided fields */
   private fun innerTypes(fields: List<Field>, fragments: List<Fragment>): List<TypeSpec> =
       fields.filter(Field::isNonScalar)
-          .map { build(it.normalizedName(), it.fields!!, it.referencedFragments(fragments),
-              it.inlineFragments ?: emptyList()) }
+          .map { build(it.normalizedName(), it.fields!!, fragments, it.inlineFragments ?: emptyList()) }
 
-  private fun inlineFragmentsTypeSpecs(inlineFragments: List<InlineFragment>): List<TypeSpec> =
-      inlineFragments.map { it.toTypeSpec() }
+  private fun inlineFragmentsTypeSpecs(inlineFragments: List<InlineFragment>, fragments: List<Fragment>): List<TypeSpec> =
+      inlineFragments.map {
+        TypeSpec.interfaceBuilder(it.interfaceName())
```

Comment:
sounds like you should still be calling `InlineFragment.toTypeSpec()` here, no?
I don't quite understand why you had to change this